bundles/org.openhab.binding.bosesoundtouch/src/main/resources/OH-INF/thing/SoundTouchWirelessLinkAdapter.xml

```diff
@@ -7,6 +7,7 @@
 	<thing-type id="wirelessLinkAdapter">
 		<label>Bose SoundTouch Wireless Link Adapter</label>
 		<description>Bose SoundTouch Wireless Link Adapter</description>
+		<semantic-equipment-tag>Router</semantic-equipment-tag>
```

Comment:
Hi. The "Link Adapter" is an audio sender/receiver. I think `AudioVisual` is a better match than `Router` (NetworkAppliance). The purpose of SoundTouch is to stream and synchronize music between multiple devices.

Reply:
Done